### ISSUE TITLE
Artist artwork doesn't load with certain albums

### ISSUE THREAD

**Issue by bbsixzz** [navidrome/navidrome#2137]
### Description

The artist artwork doesn't load when albums have an ID3 Album tag containing `:`

I copied `Play: The B Sides` with the corresponding MP3s and changed the `Artist` value from `Moby` to `80KIDZ`.

```
bbsixzz@debian:~/music/80KIDZ$ l
total 884K
drwxr-xr-x 2 root   root   4.0K Feb  2 05:28 'Play: The B Sides'
drwxr-xr-x 2 root   root   4.0K Dec 17  2021 'This Is My Shit'
-rw-r--r-- 1 root   root   873K Feb  2 05:13  artist.png
```

If I create an empty folder named `Play: The B Sides` the artwork loads.

When I remove the offending album the artwork loads.

**Comment by deluan** [comment]
Thanks for reporting. The issue is with the `:`, that is the OS path separator. I will try to fix this for the upcoming bug release fix. The only workaround is to remove the `:` from the folder and file names and do a rescan.

**Comment by deluan** [comment]
Fixed in the latest dev build. It will be part of the bug fix release 0.49.1. A full rescan will be required.
